### PR TITLE
PaloAltoNetworks Integrations overview

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Related Issues
reference: https://github.com/demisto/etc/issues/44390

## Description
After the success of https://xsoar.pan.dev/docs/reference/articles/MS_Azure_Integrations, this PR aim to give a short overview and an entry point to PANW shop customers/SAs that maps to the integrated solutions we offer.

### CODE REVIEW

**Review by Hruuttila — CHANGES_REQUESTED** [review]
Feeds are missing the Unit42 Atom feed

File: content-repo/extra-docs/articles/paloaltonetworks-integrations-overview [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+
+## Feeds
+
+### Use Cases:
+Fetch Indicators from Palo Alto Networks services.
+
+### [Unit42 ATOMs Feed](https://xsoar.pan.dev/docs/reference/integrations/unit42v2-feed)
+Unit42 feed of published IOCs, which contains known malicious indicators.
+
```
Comment (Hruuttila) [inline content-repo/extra-docs/articles/paloaltonetworks-integrations-overview:131]: Unit 42 Intel Objects Feed is also missing.

File: content-repo/extra-docs/articles/paloaltonetworks-integrations-overview [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+- Query the following PAN-OS log types: traffic, threat, url, data-filtering, and Wildfire.
+- Manage External Dynamic Lists (EDLs).
+
+### Playbooks
+- [PAN-OS Query Logs For Indicators](https://xsoar.pan.dev/docs/reference/playbooks/pan-os-query-logs-for-indicators)
+- [PAN-OS Commit COnfiguration](https://xsoar.pan.dev/docs/reference/playbooks/pan-os-commit-configuration)
+
+
+## [PAN-OS EDL Management](https://xsoar.pan.dev/marketplace/details/PaloAltoNetworks_PAN_OS_EDL_Management)
+
```
Comment (Hruuttila) [inline content-repo/extra-docs/articles/paloaltonetworks-integrations-overview:32]: Palo Alto Networks PAN-OS EDL Service is missing

Comment (Hruuttila) [reply content-repo/extra-docs/articles/paloaltonetworks-integrations-overview:32]: And EDL management is deprecated.

**Review by Hruuttila — CHANGES_REQUESTED** [review]

File: content-repo/extra-docs/articles/paloaltonetworks-integrations-overview (hunk 2 above)
Comment (bakatzir) [reply content-repo/extra-docs/articles/paloaltonetworks-integrations-overview:32]: correct, I will mention the edl service, and will rm edl management.

**Comment by ShirleyDenkberg** [comment]
@Arsenikr, @meital-aizen, @DanilVil, @Itay4, @Hruuttila Doc review completed.
@bakatzir FYI The following links lead to pages that do not provide any useful information:
- The Palo Alot Networks PAN-OS EDL Service page (https://xsoar.pan.dev/marketplace/details/EDL) 
- The Palo Alto Networks BPA page (https://xsoar.pan.dev/marketplace/details/BPA) 
- The Palo Alto Networks Threat Vault page (https://xsoar.pan.dev/marketplace/details/PaloAltoNetworks_Threat_Vault) 
- The Cortex Data Lake page(https://xsoar.pan.dev/marketplace/details/CortexDataLake)

**Comment by bakatzir** [comment]
> @Arsenikr, @meital-aizen, @DanilVil, @Itay4, @Hruuttila Doc review completed. @bakatzir FYI The following links lead to pages that do not provide any useful information:
> 
> * The Palo Alot Networks PAN-OS EDL Service page (https://xsoar.pan.dev/marketplace/details/EDL)
> * The Palo Alto Networks BPA page (https://xsoar.pan.dev/marketplace/details/BPA)
> * The Palo Alto Networks Threat Vault page (https://xsoar.pan.dev/marketplace/details/PaloAltoNetworks_Threat_Vault)
> * The Cortex Data Lake page(https://xsoar.pan.dev/marketplace/details/CortexDataLake)

I was contemplating if I should link to the integration or to the pack, but decided on the pack.
this way the reader can see not only the relevant integration, but all of the related bundled content, e.g, incident types, incident fields, classifiers, mappers.

**Review by Hruuttila — APPROVED** [review]

**Comment by bakatzir** [comment]
@reutshal I could not find my article on https://xsoar.pan.dev/docs/reference/articles
do you know what I've missed?

**Comment by reutshal** [comment]
@bakatzir Might be related to the fact that you are missing the".md" extension in your file.
I have an open PR, I'll try to fix it there - https://github.com/demisto/content-docs/pull/891
Please note that you have the option to see your changes in the preview mode, worth using it :) (I see it is missing there also)

**Pull request merged**